packages/@react-types/tooltip/src/index.d.ts

```diff
@@ -14,43 +14,31 @@ import {AriaLabelingProps, DOMProps, StyleProps} from '@react-types/shared';
 import {OverlayTriggerProps, PositionProps} from '@react-types/overlays';
 import {ReactElement, ReactNode} from 'react';
 
-export interface TooltipTriggerProps extends OverlayTriggerProps {
+export interface TooltipTriggerProps {
   /**
    * Whether the tooltip should be disabled, independent from the trigger.
    */
   isDisabled?: boolean,
 
   /**
-   * The delay time for the tooltip to show up. [See guidelines](https://spectrum.adobe.com/page/tooltip/#Immediate-or-delayed-appearance).
-   */
-  delay?: number,
-
-  /**
-   * The additional offset applied along the main axis between the element and its
-   * anchor element.
-   * @default 7
-   */
-  offset?: number,
-
-  /**
-   * The additional offset applied along the cross axis between the element and its
-   * anchor element.
-   * @default 0
+   * By default, opens for both focus and hover. Can be made to open only for focus.
    */
-  crossOffset?: number,
+  trigger?: 'focus'
+}
 
+export interface TooltipTriggerStateProps extends OverlayTriggerProps {
   /**
-   * By default, opens for both focus and hover. Can be made to open only for focus.
+   * The delay time for the tooltip to show up. [See guidelines](https://spectrum.adobe.com/page/tooltip/#Immediate-or-delayed-appearance).
+   * @default 1500
    */
-  trigger?: 'focus'
+  delay?: number
```

Comment:
Did you move this to its own type because it's unused in the aria hook? Usually we have 3 types: base stately props, stately + aria props, stately + aria + spectrum. This is diverging a bit from that pattern. Given that the prop is optional, is there any harm in including it as an option to aria too?

Reply:
> Did you move this to its own type because it's unused in the aria hook? 

Most likely - yes.

> Given that the prop is optional, is there any harm in including it as an option to aria too?

No great harm but it was hard for me to reason about the structure of those various hooks to see how they are supposed to be interconnected when they have been using types with unrelated properties. THis is the least important part of this PR though so I'm OK with reverting this change - just please confirm that you'd like me to do so.

Reply:
For now let's keep the existing pattern of the types building on one another. We can think about whether we want to change that separately.

Reply:
done